mne_bids/tests/conftest.py

```diff
@@ -21,7 +21,8 @@
 @pytest.fixture(scope="session")
 def _bids_validate():
     """Fixture to run BIDS validator."""
-    vadlidator_args = ["--config.error=41"]
+    # vadlidator_args = ["--config.error=41"]
```

Comment:
Was this really https://github.com/bids-standard/bids-validator/blob/690a34a5a723b8b436f19d9bc4da3b7a2acfaad4/bids-validator/utils/issues/list.js#L225-L229 ?
```
  41: {
    key: 'NIFTI_UNIT',
    severity: 'warning',
    reason:
      "NIfTI file's header field for unit information for x, y, z, and t dimensions empty or too short",
```

Reply:
ok, got it. This was related to:

- https://github.com/mne-tools/mne-bids/issues/238

fixed in:

- https://github.com/mne-tools/mne-bids/pull/241